src/client/app/containers/view.mode.ts

```diff
@@ -93,4 +125,12 @@ export class ViewMode implements OnInit, OnDestroy {
     feedback() {
         window.open(environment.current.config.feedbackUrl);
     }
+
+    openInPlayground(isDownload: boolean) {
+        this._store.dispatch(new Snippet.OpenInPlaygroundAction({ type: this.viewType, id: this.viewId, isDownload: isDownload }));
+    }
+
+    openInGithub() {
+        window.open(`https://gist.github.com/${this.snippet.gist}`);
```

Comment:
We use this URL in other places, could we have a helper function that generates the gist url.